src/fragments/lib/auth/js/emailpassword.mdx

```diff
@@ -51,10 +113,34 @@ async function resendConfirmationCode() {
   }
 }
 ```
+</Block>
+</BlockSwitcher>
+
 
 ### Confirm sign up
 
 If you enabled multi-factor auth, confirm the sign-up after retrieving a confirmation code from the user.
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { Auth } from 'aws-amplify';
+
+interface ConfirmSignUpParams {
+  username: string;
+  code: string;
+}
+
+export async function confirmSignUp(confirmSignUpParams: ConfirmSignUpParams) {
```

Comment:
Can you update this to include passing in `username` and `code` as a destructured object such as below:

`confirmSignUp({username, code}:ConfirmSignUpParams)`